test/_tools.font-size.scss

```diff
@@ -0,0 +1,76 @@
+@import "true";
+
+@import "../settings/settings.core";
+@import "../tools/tools.font-size";
+
+
+@mixin test-inuit-old-ie-before() {
+  $inuit-oldIE: true !global;
+}
+
+
+@include test-module("[mixin] inuit-font-size") {
+
+  @include test("it should output a font-size and line-height based on a given input") {
+
+    @include assert() {
+
+      @include output {
+        @include inuit-font-size(16px);
+      }
+
+      @include expect {
+        font-size: 1rem;
+        line-height: 1.5;
+      }
+    }
+  }
+
+  @include test("it should output a font-size and line-height when specifying a line-height") {
+
+    @include assert() {
+
+      @include output {
+        @include inuit-font-size(18px, 2);
+      }
+
+      @include expect {
+        font-size: 1.125rem;
+        line-height: 2;
+      }
+    }
+  }
+
+  @include test("it should output a font-size in pixels when declaring as important") {
```

Comment:
The explanation is not correct. It should not output font-size in pixels. It should output `!important` as part of the value of `font-size` and `line-height` properties. Does it make sense? Sorry if I'm not explaining correctly.